typings/custom-error-generator/custom-error-generator.d.ts

```diff
@@ -0,0 +1,7 @@
+
+declare module "custom-error-generator" {
```

Comment:
@thmiceli Can you PR this typing files to [DefinitelyTyped](https://github.com/borisyankov/DefinitelyTyped) so we can use `tsd` to get it?